core/deployment/src/main/java/io/quarkus/deployment/util/ReflectUtil.java

```diff
@@ -1,22 +1,17 @@
 package io.quarkus.deployment.util;
 
-import java.lang.reflect.AnnotatedElement;
-import java.lang.reflect.Array;
-import java.lang.reflect.Field;
-import java.lang.reflect.GenericArrayType;
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Member;
-import java.lang.reflect.Parameter;
-import java.lang.reflect.ParameterizedType;
-import java.lang.reflect.Type;
-import java.lang.reflect.UndeclaredThrowableException;
+import io.quarkus.deployment.annotations.BuildProducer;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.lang.reflect.*;
```

Comment:
We don't use star imports, make sure to use the IDE config file as described in https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#ide-config-and-code-style

Reply:
You were right @gastaldi -the IDE config file wasn't included. Now it is, however the wildcard import is still there :(